federator/.gitignore

```diff
@@ -15,3 +16,6 @@ db/*
 !db/README.md
 heartbeat.log
 federator.log
+.env
+.env.local
+.env.prod
```

Comment:
maybe we can ignore this in the top-level .gitignore so it works for both the bridge and federator folders? [source](https://stackoverflow.com/a/29260747/3922574)

Reply:
sure